.github/workflows/sonarqube.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   build:
     name: Build
-    runs-on: windows-latest
+    runs-on: windows-2022
```

Comment:
Bah. SonarQube. I don't think we run this  anymore. 

Unless you find otherwise, I think we should delete this file.